en/docs/administer/key-managers/configure-auth0-connector.md

```diff
@@ -0,0 +1,302 @@
+                This is not mandatory since the Token Validation Method cannot be <b>introspect</b> for Auth0</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>Self Validate JWT</td>
+            <td>The <code>kid</code> value is used to validate the JWT token signature. If the `<code>kid</code> value is not present, the <code>gateway_certificate_alias</code> is used.</td>
+            <td>Optional</td>
+        </tr>
+        <tr class="odd">
+            <td>Use introspect</td>
+            <td>Cannot use this option since Auth0 does not expose an introspection endpoint</td>
+            <td>Optional</td>
+        </tr>
+        <tr class="even">
+            <td><b>Token Handling Options</b></td>
+            <td>This provides a way to validate the token for this particular authorization server.</td>
+            <td>Optional. 
+                This is mandatory if the Token Validation Method is <b>introspect</b></td>
+        </tr>
+        <tr class="odd">
+            <td>REFERENCE</td>
+            <td>The tokens that match a specific regular expression (regEx) are validated.</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td>JWT</td>
+            <td>The tokens that match a specific JWT is validated.</td>
+            <td></td>
+        </tr>
+        <tr class="odd">
+            <td>CUSTOM</td>
+            <td>The tokens that match a custom pattern are validated.</td>
+            <td></td>
+        </tr>
+        <tr class="even">
+            <td><b>Claim Mappings</b></td>
+            <td>Local and remote claim mapping.</td>
+            <td>Optional</td>
+        </tr>
+    </table>    
+
+5.  Finally, you can save the configs.
+
+## Step 3 - Generate keys using the Auth0 Key Manager
+
+1.  Create new application from the developer portal.
+
+    [![auth0 dev app create]({{base_path}}/assets/img/administer/auth0-dev-app-create.png)]({{base_path}}/assets/img/administer/auth0-dev-app-create.png)    
+
+2.  Then click either production or sandbox, Select Auth0 fill the relevant fields accordingly.
+
+    [![auth0 app creation form]({{base_path}}/assets/img/administer/auth0-app-creation-form.png)]({{base_path}}/assets/img/administer/auth0-app-creation-form.png)    
+    
+    *Please note that Audience of the API field is mandatory to generate an access token for Auth0. Therefore, please provide it when the application keys generating.*
+    
+    You can get the audience of the api by checking the API.
+    
+    [![auth0 resource api]({{base_path}}/assets/img/administer/auth0-resource-api.png)]({{base_path}}/assets/img/administer/auth0-resource-api.png)    
+
+3.  Once the keys generated, It will reflect in the UI.
+    
+    [![auth0 created app]({{base_path}}/assets/img/administer/auth0-created-app.png)]({{base_path}}/assets/img/administer/auth0-created-app.png)        
+    
+5. Finally, token will be generated successfully.
```

Comment:
As we are numbering the instructions, we should not use terms like "Initially", "Then", "Finally" at the start of the instructions.